README.md

```diff
@@ -1,2 +1,5 @@
 # k8s.io
+
 Kubernetes files for various sites and infrastructure.
+
+Please see https://github.com/kubernetes/community/tree/master/wg-k8s-infra for more information
```

Comment:
tiny nit: let's use the git.k8s.io link here?

Reply:
done